make/migrations/postgresql/0030_2.0.0_schema.up.sql

```diff
@@ -191,6 +191,8 @@ DROP TABLE IF EXISTS access_log;
 /*remove the constraint for project_id in table 'notification_policy'*/
 ALTER TABLE notification_policy DROP CONSTRAINT unique_project_id;
 
+/*the existing policy has no name, to make sure the unique constraint for name works, use the id as name*/
+UPDATE notification_policy SET name=id;
```

Comment:
Will the name of the policy be displayed in the portal ? If so `ploicy-{id}` will be better than `{id}`.

Reply:
ok